storage/persistent_storage/persistent-storage-ocs.adoc

```diff
@@ -8,6 +8,8 @@ Red Hat OpenShift Container Storage is a provider of agnostic persistent storage
 
 Red Hat OpenShift Container Storage provides its own documentation library. The complete set of Red Hat OpenShift Container Storage documentation identified below is available at https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.6/
 
+OpenShift Container Storage on top of RHHI for Virtualization (hyperconverged nodes that host virtual machines installed with Red Hat OpenShift Container Platform) is not a supported configuration.
```

Comment:
Couple of suggestions here to conform with ISG and OCP docs style:
- s/RHHI for Virtualization/Red Hat Hyperconverged Infrastructure (RHHI) for Virtualization
- s/Red Hat OpenShift Container Platform/{product-title}
- Avoid parentheses if possible. Suggest using commas and rewording to something like this: 
> OpenShift Container Storage on top of Red Hat Hyperconverged Infrastructure (RHHI) for Virtualization, which uses hyperconverged nodes that host virtual machines installed with {product-title}, is not a supported configuration.

Also, we now have attributes for OCS in the `modules/common-attributes.adoc` file: 
- :rh-storage-first: Red Hat OpenShift Container Storage
- :rh-storage: OpenShift Container Storage
This ^ is out of scope probably for this PR as it would need to be applied beyond just this sentence, but wanted to make you aware.